### PR TITLE
Change streaming iot tests to start up tonic server

### PR DESCRIPTION
**Pull request by bbalser**

### CODE REVIEW

**Review by andymck — APPROVED** [review]

File: ingest/tests/iot_ingest.rs [hunk 1]
```diff
@@ -0,0 +1,576 @@
...
+        FileSinkClient {
+            sender: tx,
+            metric: "metric",
+        },
+        MockFileSinkReceiver { receiver: rx },
+    )
+}
+
+async fn connect_and_stream(port: u64) -> TestClient {
+    let mut client = (|| PocLoraClient::connect(format!("http://127.0.0.1:{}", port)))
```
Comment (jeffgrunewald) [inline ingest/tests/iot_ingest.rs:420]:
```suggestion
    let mut client = (|| PocLoraClient::connect(format!("http://127.0.0.1:{port}")))
```

**Review by jeffgrunewald — APPROVED** [review]

**Review by maplant — APPROVED** [review]
This looks good. My one comment is that all of these tests are structurally largely the same, so perhaps a macro would be a good way to cut down on the boiler plate.

File: ingest/src/server_iot.rs [hunk 2]
```diff
@@ -139,6 +139,7 @@ impl StreamState {
         loop {
             tokio::select! {
                 _ = tokio::time::sleep_until(self.timeout) => {
+                    tracing::debug!("stream request timed out");
```
[the remark this answers is not in the document]
Comment (bbalser) [reply ingest/src/server_iot.rs]: Addressed 36e84b3

File: ingest/tests/iot_ingest.rs (hunk 1 above)
Comment (bbalser) [reply ingest/tests/iot_ingest.rs]: Fixed 36e84b3

**Pull request merged**